assets/js/blocks/active-filters/active-attribute-filters.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { useCollection, useQueryStateByKey } from '@woocommerce/base-hooks';
+
+/**
+ * Internal dependencies
+ */
+import { renderItem } from './utils';
+import { removeAttributeFilterBySlug } from '../../utils/attributes-query';
+
+/**
+ * Component that renders active attribute (terms) filters.
+ */
+const ActiveAttributeFilters = ( { attributeObject = {}, slugs = [] } ) => {
+	const { results, isLoading } = useCollection( {
+		namespace: '/wc/store',
+		resourceName: 'products/attributes/terms',
+		resourceValues: [ attributeObject.id ],
+	} );
+
+	const [ productAttributes, setProductAttributes ] = useQueryStateByKey(
+		'product-grid',
+		'attributes',
+		[]
+	);
+
+	if ( isLoading ) {
+		return null;
+	}
+
+	const attributeLabel = attributeObject.label;
+
+	return slugs.map( ( slug ) => {
```

Comment:
Nice!  Seeing things like this though are making me want to implement some performance testing in our repo.  That way we can try different variations to see what works better (for example, `useMemo` wrapping the component generation here might be a good fit if the number of slugs is large, but if they are typically a smaller number maybe less so).

We could also try wrapping the functional component in `React.memo` (should be exposed on `wp.element` as well). 

I don't think you need to add that right now though, it might be pre-optimization at this point.

Reply:
so...do nothing? :p 

In the block the usage of this component does use useMemo so there is some caching happening.

Reply:
> so...do nothing?

Yup, sometimes I'll leave observational feedback because it might trigger additional insight for another reviewer or in the original author.  So unless my reviews make an explicit action request it's up to the author whether they implement or not (and I'll usually indicate whether I think it should be actioned on).